Added mobile sdks features for chat (#70817)
Azure/azure-content: articles/communication-services/concepts/chat/sdk-features.md

| Core Capabilities | Create a chat thread between 2 or more users (up to 250 users)                                                       | ✔️   | ✔️  | ✔️    | ✔️   |  ✔️    | ✔️   |
|                   | Update the topic of a chat thread                                                                              | ✔️   | ✔️ | ✔️    | ✔️   |  ✔️    | ✔️   |
|                   | Add or remove participants from a chat thread                                                                           | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |
|                   | Choose whether to share chat message history with the participant being added                                   | ✔️   | ✔️   | ✔️    | ✔️  |  ✔️    | ✔️   |
|                   | Get a list of participants in a chat thread                                                                          | ✔️   | ✔️  | ✔️ | ✔️ |  ✔️    | ✔️   |
|                   | Delete a chat thread                                                                                              | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |

> |                   | Given a communication user, get list of chat threads the user is part of                                           | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |

Reviewer: ```suggestion
|                   | Given a communication user, get the list of chat threads the user is part of                                           | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |
```

|                   | Edit the content of a sent message                                                                                | ✔️   | ✔️  | ✔️ | ✔️ |  ✔️    | ✔️   |
|                   | Delete a message                                                                                                       | ✔️   | ✔️  | ✔️ | ✔️ |  ✔️    | ✔️   |
|                   | Read receipts for messages that have been read by other participants in a chat <br/> *Not available when there are more than 20 participants in a chat thread*    | ✔️   | ✔️  | ✔️    | ✔️   |  ✔️    | ✔️   |
|                   | Get notified when participants are actively typing a message in a chat thread <br/> *Not available when there are more than 20 members in a chat thread*      | ✔️   | ✔️   | ✔️    | ✔️    |  ✔️    | ✔️   |
|                   | Get all messages in a chat thread <br/>                                                                         | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |
|                   | Send unicode emojis as part of message content                                                                            | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |

> |Real-time signaling (enabled by proprietary signalling package**)|  Subscribe to get real-time updates for incoming messages & other operations in your chat app. To see a list of supported updates, refer to chat concepts [here](https://docs.microsoft.com/en-us/azure/communication-services/concepts/chat/concepts#real-time-signaling)                                     | ✔️   | ❌    | ❌  | ❌  | ❌  | ❌  |

Reviewer: ```suggestion
|Real-time signaling (enabled by proprietary signaling package**)|  Subscribe to get real-time updates for incoming messages & other operations in your chat app. To see a list of supported updates, refer to chat concepts [here](https://docs.microsoft.com/en-us/azure/communication-services/concepts/chat/concepts#real-time-signaling)                                     | ✔️   | ❌    | ❌  | ❌  | ❌  | ❌  |
```

| Event Grid support             | Use the integration with Azure Event Grid and configure your communication service to execute business logic based on chat activity or to plug in a custom push notification service   | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |
| Monitoring        | Use the API request metrics emitted Azure portal to build dashboards, monitor health of your chat app, and set alerts to detect abnormalities      | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |

> |                   | Configure your ACS resource to recieve chat operational logs for monitoring and diagnostic purposes          | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |

Reviewer: ```suggestion
|                   | Configure your Communication Services resource to receive chat operational logs for monitoring and diagnostic purposes          | ✔️   | ✔️  | ✔️    | ✔️  |  ✔️    | ✔️   |
```
